kfs-core/src/main/java/edu/arizona/kfs/sys/batch/service/impl/RoomImportServiceImpl.java

```diff
@@ -339,59 +318,57 @@ private void disableKFSRoomsNotInArchibus(PrintWriter outReportWriter, Room kfsR
 		keys.put("buildingCode", buildingCode);
 		archRoom = (ArchibusRooms) businessObjectService.findByPrimaryKey(ArchibusRooms.class, keys);
-		
+
 		if (ObjectUtils.isNull(archRoom) && !KFSParameterKeyConstants.NO_ROOM_LOCATION.equals(kfsRoom.getBuildingRoomNumber())) {
 			if (kfsRoom.isActive()) {
-				LOG.debug("inactivating building in kfs that does not exist in Archibus");
+				LOG.info("inactivating building in kfs that does not exist in Archibus");
 				kfsRoom.setActive(false);
-				businessObjectService.save(kfsRoom);
+				 businessObjectService.save(kfsRoom);
 				writetoReportOutputFile(outReportWriter, reportMsg, kfsRoom.getCampusCode(), kfsRoom.getBuildingCode(), kfsRoom.getBuildingRoomNumber(), "inactivating room in kfs that does not exist in Archibus");
 			}
 		}
 	}
-	
+
 	private PrintWriter setupErrorReportOutputFiles(PrintWriter outErrorReportWriter) {
 		try {
 			String outputErrorFile = reportDirectoryName + File.separator + "roomImportErrorReport_" + dateTimeService.toDateTimeStringForFilename(dateTimeService.getCurrentDate()) + ".txt";
 			outErrorReportWriter = new PrintWriter(new BufferedWriter(new FileWriter(outputErrorFile)));
-		}
+		} 
 		catch (FileNotFoundException e) {
 			LOG.error("Error in RoomImportServiceImpl in method setUp prepareRoomImport. File won't process");
 			throw new RuntimeException(e);
-		}
+		} 
 		catch (IOException e) {
 			LOG.error("Error in RoomImportServiceImpl in method setUp prepareRoomImport. IOException");
 			throw new RuntimeException(e);
-		}
+		} 
 		catch (Exception e) {
 			LOG.error("Error in RoomImportServiceImpl in method setUp prepareRoomImport. Exception");
 			throw new RuntimeException(e);
 		}
-		LOG.debug("Exit prepareRoomImport() " + System.currentTimeMillis());
 		return outErrorReportWriter;
 	}
-	
+
 	private PrintWriter setupReportOutputFiles(PrintWriter outReportWriter) {
 		try {
-			String outputFile = reportDirectoryName + File.separator + "roomImportSuccessReport_" + dateTimeService.toDateStringForFilename(dateTimeService.getCurrentDate()) + ".txt";
+			String outputFile = reportDirectoryName + File.separator + "roomImportSuccessReport_" + dateTimeService.toDateTimeStringForFilename(dateTimeService.getCurrentDate()) + ".txt";
 			outReportWriter = new PrintWriter(new BufferedWriter(new FileWriter(outputFile)));
-		}
+		} 
 		catch (FileNotFoundException e) {
 			LOG.error("Error in RoomImportServiceImpl in method setUp prepareRoomImport. File won't process");
 			throw new RuntimeException(e);
-		}
+		} 
 		catch (IOException e) {
 			LOG.error("Error in RoomImportServiceImpl in method setUp prepareRoomImport. IOException");
 			throw new RuntimeException(e);
-		}
+		} 
 		catch (Exception e) {
 			LOG.error("Error in RoomImportServiceImpl in method setUp prepareRoomImport. Exception");
 			throw new RuntimeException(e);
 		}
-		LOG.debug("Exit prepareRoomImport() " + System.currentTimeMillis());
```

Comment:
Same as above.  This debug statement could be helpful if there are batch failures.